js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var Utils = require('../utils');
+var classnames = require('classnames');
+var CONSTANTS = require('../../constants/constants');
+var Tab = require('./tab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var ClosedCaptionMultiAudioMenu = React.createClass({
+  componentDidMount: function() {
+    var multiAudioCol = ReactDOM.findDOMNode(this.refs.multiAudioCol);
+    var closeCaptionsCol = ReactDOM.findDOMNode(this.refs.closeCaptionsCol);
+    if (multiAudioCol && closeCaptionsCol && typeof Utils.getMaxElementWidth === 'function') {
+      var maxWidth = Utils.getMaxElementWidth(multiAudioCol, closeCaptionsCol);
+      multiAudioCol.style.width = maxWidth + 'px';
```

Comment:
Is this code meant to divide each element into equal size columns? We should be able to do this with flexbox in CSS. Also since this is done in `componentDidMount` it looks like it won't react to style changes.

Reply:
Good point, we removed that method and replaced it with flexbox implementation.